src/network/networkd-link.c

```diff
@@ -1309,7 +1309,7 @@ static int link_set_proxy_arp(Link *link) {
         return 0;
 }
 
-static int link_configure_after_setting_mtu(Link *link);
+static int link_configure_continue(Link *link);
```

Comment:
The name of the function because we got into a loop after setting mtu . The ethernet card reset and  gets into a loop.

Reply:
@ddstreet Could you briefly mention that why we configure links in two steps in `link_configure_continue()`? Some drivers reset the interface when its MTU is changed. So without doing so, the networkd entered infinite loop.

Reply:
I added a comment right above the function and re-pushed - let me know if I should adjust it or move it?  Thanks!

Reply:
Thank you!